### DISCUSSION TITLE
How to prevent DB from generating new class images?

### DISCUSSION THREAD

**Discussion by wktra** [d8ahazard/sd_dreambooth_extension#729] [Q&A]
Took a LONG break from DB.

I've come back and I can understand most of the changes.

But now training keeps generating class images, even though I provided a folder with my own classification images.

How do I prevent this? Please help.

**Comment by BigBlueGX** [comment]
Normally I think that you set it at the Concepts area, when creating a new model.
At "Image Generation" ->"Class Images Per Instance Image" - > set it to 0

**Reply by wktra** [reply]
@BigBlueGX I get this error: _Exception training model: ''NoneType' object has no attribute 'items''._

**Comment by moraaaaaan** [comment]
> Took a LONG break from DB.
> 
> I've come back and I can understand most of the changes.
> 
> But now training keeps generating class images, even though I provided a folder with my own classification images.
> 
> How do I prevent this? Please help.

have you solve this? it's driving me crazy...

**Reply by Gaspa79** [reply]
I'm having the same problem
